hudi-common/src/main/java/org/apache/hudi/common/model/EngineType.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hudi.common.model;
+
+/**
+ * Hoodie data processing engine. support only Apache Spark and Apache Flink for now.
+ */
+public enum EngineType {
```

Comment:
Can we put this defination into `hudi-client-common`?

Reply:
@yanghua `EngineType` is an attribute of `DefaultHoodieConfig`, should be place in `hudi-common`

Reply:
> Can we put this definition into `hudi-client-common`?

please take another look when free